pkg/transaction/sender_matcher.go

```diff
@@ -71,35 +71,62 @@ func (m *Matcher) Matches(ctx context.Context, tx []byte, networkID uint64, send
 		return nil, ErrGreylisted
```

Comment:
Do we use this `Timestamp` value anywhere? I don't see a greylisting mechanism anywhere?

Reply:
Timestamp is set to `time.Now` on error. Then this if will cause `ErrGreylisted` for 5 minutes.